hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ledger/accounts/ContractCustomizer.java

```diff
@@ -99,13 +108,15 @@ public static ContractCustomizer fromHapiCreation(
      * @return an appropriate child customizer
      */
     public static ContractCustomizer fromSponsorContract(
-            final AccountID sponsor, final TransactionalLedger<AccountID, AccountProperty, HederaAccount> ledger) {
+            final AccountID sponsor,
+            final TransactionalLedger<AccountID, AccountProperty, HederaAccount> ledger,
+            final Address customizerAppliesToAddress) {
```

Comment:
Q: this is the address of the contract to be created? 
Presumably this address is the result of id to long zero or CREATE2 right? Or is it a smaller set of options?

Reply:
Yes, the new address where we should be creating the contract.